src/blocks/call-to-action/settings.js

```diff
@@ -21,7 +21,7 @@ import ResponsiveSlider from '@Components/responsive-slider';
 import { useDeviceType } from '@Controls/getPreviewType';
 
 const Settings = ( props ) => {
-	props = props.parentProps;
+
 	const { setAttributes, attributes } = props;
 	const deviceType = useDeviceType();
```

Comment:
```suggestion
	const { setAttributes, attributes, deviceType } = props;
```
and also remove importing

Reply:
Removed